docs/layers.md

```diff
@@ -262,6 +262,7 @@ _TIP: Some `landuse` features only exist as point features in OpenStreetMap. Fin
 
 * `protect_class`: Common values include `1`, `2`, `3`, `4`, `5`, `6`. See [OSM wiki](https://wiki.openstreetmap.org/wiki/Tag:boundary%3Dprotected_area#Protect_classes_for_various_countries) for more information.
 * `operator`: e.g. `United States National Park Service`, `United States Forest Service`, `National Parks & Wildlife Service NSW`.
+* `scalerank`: An integer from 1 to 6. If this property is set on a feature at a particular zoom then it indicates that this feature is an appropriate size to display at this zoom. Features without this property set may be too small, or missing "importance raising" properties.
```

Comment:
Looks like this is [written](https://github.com/tilezen/vector-datasource/pull/980/files#diff-e736294345a150d2b5eed04d43d517c4R2) as `3` to `16` (but could conceivable be set to 0 to 16)?